### DISCUSSION TITLE
Account Got Voided.

### DISCUSSION THREAD

**Discussion by DozenLimeStone** [community/community#83279] [New to GitHub] [answered]
Recently I posted a discussion on other account, but if I switch to the main account and refresh, it says 404 discussion not found.
Also on other account I cannot cast votes, my comments are just "private", nobody can see them, personal access token gives only 60 rate limit on Rest API instead of 5000... A lot of things.

### What happened?

**Comment by queenofcorgis** [comment]
Hi there @DozenLimeStone, 

Was this discussion in our community or elsewhere? Would you mind sharing the username as well so we can better assist?

**Reply by DozenLimeStone** [reply]
@queenofcorgis Thanks for responding, I didn't actually expect that 🤣.
Yes, the discussion that I opened on that account was in API part of discussions, the username is @IDKMYSURNAMEWASLEAKED
Pretty funny name I picked.

**Reply by queenofcorgis** [reply, marked as answer]
@DozenLimeStone thanks for your patience, we checked some things on our end and now your content should be appearing. I can confirm I can see your original discussion post! Great username by the way 😃

**Reply by DozenLimeStone** [reply]
@queenofcorgis Thanks!
